plugins/graphiql/src/components/GraphiQLPage/GraphiQLPage.tsx

```diff
@@ -14,14 +14,8 @@
  * limitations under the License.
  */
 import React from 'react';
-import {
-  Content,
-  Header,
-  HeaderLabel,
-  Page,
-  Progress,
-  useApi,
-} from '@backstage/core';
+import { Content, Header, HeaderLabel, Page, Progress } from '@backstage/core';
```

Comment:
components!

Reply:
Not aiming to move over all existing usage with this PR, just moved over a few things to make sure stuff works.